jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UISelectAssert.java

```diff
@@ -23,167 +23,220 @@
     }
+    
     @JDIAction("Assert that '{name}' enabled items {0}")
-    public A enabled(Matcher<? super List<String>> condition) {
-        jdiAssert(element.listEnabled(), condition);
+    public A enabled(Matcher<? super List<String>> condition, String... messages) {
+        jdiAssert(element.listEnabled(), condition, messages);
         return (A) this;
     }
+    
+    /**
+     * This method do not accept custom error messages.
+     * @param enabled
+     * @return
+     */
     public A enabled(String... enabled) {
         return enabled(hasItems(enabled));
     }
+    
     @JDIAction("Assert that '{name}' disabled items {0}")
-    public A disabled(Matcher<? super List<String>> condition) {
-        jdiAssert(element.listDisabled(), condition);
+    public A disabled(Matcher<? super List<String>> condition, String... messages) {
+        jdiAssert(element.listDisabled(), condition, messages);
         return (A) this;
     }
+    
+    /**
+     * This method do not accept custom error messages.
+     * @param disabled
+     * @return
+     */
     public A disabled(String... disabled) {
         return disabled(hasItems(disabled));
     }
-
+    
     @JDIAction("Assert that '{name}' size {0}")
-    public A size(Matcher<Integer> condition) {
-        jdiAssert(element.size(), condition);
+    public A size(Matcher<Integer> condition, String... messages) {
+        jdiAssert(element.size(), condition, messages);
         return (A) this;
     }
-
+    
     /**
      * Check that the list size is given size
+     *
      * @param size to compare
      * @return DataListAssert
      */
     @JDIAction("Assert that '{name}' size {0}")
-    public A size(int size) {
-        return size(equalTo(size));
+    public A size(int size, String... messages) {
+        return size(equalTo(size), messages);
     }
+    
     @JDIAction("Assert that '{name}' size {0}")
-    public A empty() {
-        return size(0);
+    public A empty(String... messages) {
+        return size(0, messages);
     }
+    
     @JDIAction("Assert that '{name}' size {0}")
-    public A notEmpty() {
-        return size(greaterThan(0));
+    public A notEmpty(String... messages) {
+        return size(greaterThan(0), messages);
     }
+    
     @JDIAction("Assert that '{name}' is displayed")
-    public A displayed() {
-        jdiAssert(element.isDisplayed() ? "displayed" : "hidden", Matchers.is("displayed"));
+    public A displayed(String... messages) {
+        jdiAssert(element.isDisplayed() ? "displayed" : "hidden", Matchers.is("displayed"), messages);
         return (A) this;
     }
+    
     @JDIAction("Assert that '{name}' is expanded")
-    public A expanded() {
-        jdiAssert(element.list().isDisplayed() ? "expanded" : "hidden", Matchers.is("expanded"));
+    public A expanded(String... messages) {
+        jdiAssert(element.list().isDisplayed() ? "expanded" : "hidden", Matchers.is("expanded"), messages);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)